packages/manager/modules/sms/src/sms/sms/outgoing/telecom-sms-sms-outgoing.html

```diff
@@ -1,375 +1,164 @@
+        </oui-datagrid-topbar>
+        <oui-datagrid-column
+            title="'sms_sms_outgoing_label_date' | translate"
+            property="creationDatetime"
+            type="date"
+            filterable
+            sortable="desc"
+        >
+            <span
+                data-ng-bind="$row.creationDatetime | date: 'dd/MM/yyyy'"
+            ></span
+            ><br />
+            <span
+                data-ng-bind="$row.creationDatetime | date: 'HH:mm:ss'"
+            ></span>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            title="'sms_sms_outgoing_label_sender' | translate"
+            property="sender"
+            filterable
+            sortable
         >
+            <span data-ng-if="$row.sender" data-ng-bind="$row.sender"> </span>
             <span
-                class="oui-icon mr-2 oui-icon-trash_concept"
-                aria-hidden="true"
-                data-ng-if="!SmsOutgoingCtrl.outgoing.isDeleting"
+                data-ng-if="!$row.sender"
+                data-oui-tooltip="{{ ::'sms_sms_outgoing_number_allowed_response_fr_only' | translate }}"
+                data-translate="sms_sms_outgoing_number_allowed_response"
             >
+            </span
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            title="'sms_sms_outgoing_label_receiver' | translate"
+            property="receiver"
+            filterable
+            sortable
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            title="'sms_sms_outgoing_label_tariff_code' | translate"
+            property="tariffCode"
+            sortable
+        >
+            <span data-ng-bind="$row.tariffCode || '-'"></span>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            title="'sms_sms_outgoing_label_message' | translate"
+            property="message"
+            filterable
+            sortable
+        >
+            <span data-ng-if="$row.message.length >= 25">
+                <span
+                    data-ng-bind="$row.message | limitTo: 25"
+                    data-oui-tooltip="{{ $row.message }}"
+                >
+                </span
+                >…
             </span>
-            <oui-spinner
-                class="ml-2"
-                data-ng-if="SmsOutgoingCtrl.outgoing.isDeleting"
-                data-size="s"
-            >
-            </oui-spinner>
             <span
-                data-ng-pluralize
-                data-count="SmsOutgoingCtrl.getSelection().length"
-                data-when="{
-                          '0': ('sms_sms_outgoing_delete' | translate),
-                          'one': ('sms_sms_outgoing_delete' | translate),
-                          'other': ('sms_sms_outgoing_delete_plural' | translate:{ count: {} })
-                      }"
+                data-ng-if="$row.message.length < 25"
+                data-ng-bind="$row.message"
             >
             </span>
```

Comment:
Use css to display ellipsis.

Reply:
In fact i reuse the already existing. There is also a tooltip if more 25 length